test/libsolidity/semanticTests/smoke/constructor.sol

```diff
@@ -15,6 +15,6 @@ contract C {
 // ----
 // constructor(), 2 wei: 3 ->
 // state() -> 3
-// balance() -> 2
```

Comment:
Should remove the `balance()` function from this contract.

Reply:
I'll use both.